### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by mry-usds**
This PR does ...

## Changes
 [X] After logging out, was able to hit the back button and still see everything on the download index (Mac Chrome).
Nav
[X] Link "Prime Data Hub" to the download index.
Header
[X] Change Location name "pima_az_phd" to friendlier name. Maybe ex "Pima County, AZ" or "Pima County Health Department"? (Maybe that display name is something the user could edit? Something to ask partners about)
CSV lists
[X] Expires column: add plural logic to "day"/"days"
[X] Previous n days: if no data available, change text from "No data found" to "No data is available for the previous n days. If you think this might be in error, contact support@prime.cdc.gov."
[X] Add GMT to end of Date
[X] For Receiver change to friendlier description, same as H1
 Footer
[X] Add a target=_blank to the CDC link in the footer
[X] Add support email address to the footer

### CODE REVIEW

File: prime-router/assets/csv-download-site/index__inline.html [hunk 1]
```diff
@@ -18,10 +18,16 @@
             }
     
             function logout(){
-                removeItem("user")
-                removeItem("organization")
+                removeItem("jwt")
```
Comment (RickHawesUSDS) [inline prime-router/assets/csv-download-site/index__inline.html:21]: Can you explain this change?

File: prime-router/assets/csv-download-site/index__inline.html [hunk 2]
```diff
@@ -18,10 +18,16 @@
...
                 window.location.replace("http://www.cdc.gov");
             }
```
Comment (RickHawesUSDS) [inline prime-router/assets/csv-download-site/index__inline.html:23]: I thought we going to logout to the site?

File: prime-router/src/main/kotlin/azure/DownloadFunction.kt [hunk 3]
```diff
@@ -119,10 +127,10 @@ class DownloadFunction {
         val htmlTemplate: String = Files.readString(Path.of(DOWNLOAD_PAGE))
         val headers = DatabaseAccess(dataSource = DatabaseAccess.dataSource).fetchHeaders(OffsetDateTime.now().minusDays(DAYS_TO_SHOW), orgName)
 
-        System.out.println("count of headers = ${headers.size}")
+        val org = WorkflowEngine.metadata.findOrganization(orgName.replace('_', '-'))
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/DownloadFunction.kt:131]: Good.

File: prime-router/src/main/kotlin/azure/DownloadFunction.kt [hunk 4]
```diff
@@ -104,13 +109,16 @@ class DownloadFunction {
                 it.task.createdAt
             }
             .map {
+                val org = WorkflowEngine().metadata.findOrganization(orgName.replace("_", "-"))
+                val svc = WorkflowEngine.metadata.findService(it.task.receiverName)
                 TestResult(
                     it.task.createdAt.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")),
-                    it.task.receiverName,
+                    if (svc !== null) svc.description else if (org !== null) org.description else "Unknown",
                     DAYS_TO_SHOW - it.task.createdAt.until(OffsetDateTime.now(), ChronoUnit.DAYS),
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/DownloadFunction.kt:117]: good.

**Review by RickHawesUSDS — APPROVED** [review]
Look's good, but a couple of questions on logout.

File: prime-router/assets/csv-download-site/index__inline.html (hunk 1 above)
Comment (mry-usds) [reply prime-router/assets/csv-download-site/index__inline.html:21]: that removes the cookies so back doenst work and we generally dont have the jwt token lying around.

File: prime-router/assets/csv-download-site/index__inline.html (hunk 2 above)
Comment (mry-usds) [reply prime-router/assets/csv-download-site/index__inline.html:23]: we do - thats the window.location.replace() call...

**Pull request merged**